tools/nl/embeddings/README.md

```diff
@@ -9,16 +9,9 @@ There are multiple sizes of indexes: small, medium.
 As of May 2023, the small index has ~1.3K variables and medium index has 5.3K
 variables.
 
-## Curated Input Google Sheet
-
-Latest sheet as of May 2023 is
-[`DC_NL_SVs_Curated`](https://docs.google.com/spreadsheets/d/1-QPDWqD131LcDTZ4y_nnqllh66W010HDdows1phyneU).
-
-This is a common sheet across the different index sizes.
-
 ## Making a change to the embeddings.
 
-1. For any carefully curated SVs/topics, make edits to the [latest sheet](https://docs.google.com/spreadsheets/d/1-QPDWqD131LcDTZ4y_nnqllh66W010HDdows1phyneU). The columns in this sheet are:
+1. For any carefully curated SVs/topics, make edits to the [latest sheet](https://docs.google.com/spreadsheets/d/1-QPDWqD131LcDTZ4y_nnqllh66W010HDdows1phyneU). The columns in this sheet are: <a name="curated-input"></a>
```

Comment:
A bit unclear here. Why do we still need this reference to the Google sheet?

Reply:
Ah, I had changed that but somehow undid it at some point. Updated